features/step_definitions/tag_steps.rb

```diff
@@ -153,8 +153,37 @@
   end
 end
 
+Given(/^the following unsorted tag exists$/) do |table|
+  table.hashes.each do |hash|
+    FactoryGirl.create(:unsorted_tag, hash)
+  end
+end
+
+Given(/^the following typed tags exists$/) do |table|
+  table.hashes.each do |hash|
+    type = hash["type"].classify.constantize
+    hash.delete("type")
+    FactoryGirl.create(type, hash)
+  end
+end
+
 ### WHEN
 
+When /^I check the canonical wrangling tag "([^"]*)"$/ do |tagname|
+  tag = Tag.find_by_name(tagname)
+  check("canonicals_#{tag.id}")
+end
+
+When /^I select "([^"]*)" for "([^"]*)" tag$/ do |type, tagname|
```

Comment:
This step refers to selecting a tag type from the select menus on http://test.archiveofourown.org/unsorted_tags, right?

I'd suggest `I select "([^"]*)" for the unsorted tag "([^"]*)"`